lib/ajValidation/ajvValidation.js

```diff
@@ -17,6 +46,103 @@ function isPmVariable (value) {
   return _.isString(value) && _.startsWith(value, '{{') && _.endsWith(value, '}}');
 }
 
+/**
+ * Checks if value is the representation of its type like:
+ * "<integer>"
+ *
+ * @param {string} value - Value to check for
+ * @param {string} type - The type in the schemna
+ * @returns {Boolean} the value is the representation of its type
+ */
+function compareType(value, type) {
+  return value === '<' + type + '>';
+}
+
+/**
+ * Checks if value is the representation of its type like:
+ * "<integer>"
+ * Works in array types
+ * @param {string} value - Value to check for
+ * @param {*} types - The types in the schemna
+ * @returns {Boolean} the value is the representation of its type
+ */
+function isTypeValueArrayCheck(value, types) {
+  return types.find((type) => {
+    return compareType(value, type);
+  }) !== undefined;
+}
+
+/**
+ * Checks if value is the representation of its type like:
+ * "<integer>"
+ * Works in array types
+ * @param {string} value - Value to check for
+ * @param {*} types - The types in the schemna
+ * @returns {Boolean} the value is the representation of its type
+ */
+function checkValueOnlyTypes(value, types) {
+  return Array.isArray(types) ? isTypeValueArrayCheck(value, types) : compareType(value, types);
+}
+
+/**
+ * Checks if value is the representation of its type like:
+ * "<integer>"
+ * Works in array types
+ * @param {string} value - Value to check for
+ * @param {*} types - The types in the schema
+ * @param {*} format - format from the schema
+ * @returns {Boolean} the value is the representation of its type
+ */
+function checkValueTypesAndFormat(value, types, format) {
+  let typesNotInMapp = [],
+    typesArray = Array.isArray(types) ? types : [types],
+    found = typesArray.find((type) => {
+      let defaultValue;
+      if (typesMap.hasOwnProperty(type)) {
+        defaultValue = getDefaultFromTypeAndFormat(type, format);
+
+        // in case the format is a custom format (email, hostname etc.)
+        // https://swagger.io/docs/specification/data-models/data-types/#string
+        if (!defaultValue && format) {
+          defaultValue = '<' + format + '>';
+        }
+      }
+      else {
+        typesNotInMapp.push(type);
+      }
+      return defaultValue === value;
+    });
+  if (found) {
+    return true;
+  }
+
+  found = typesNotInMapp.find((type) => {
+    let defaultValue;
+    defaultValue = '<' + type + (format ? ('-' + format) : '') + '>';
```

Comment:
Can we add test that covers this flow as well?

Reply:
Hi, those scenarios are covered here https://github.com/postmanlabs/openapi-to-postman/pull/481/files#diff-5c88828172af367bb2a9b23015030930a26b586ae051830310717a0bdc4b49c5R471

and here https://github.com/postmanlabs/openapi-to-postman/pull/481/files#diff-5c88828172af367bb2a9b23015030930a26b586ae051830310717a0bdc4b49c5R464